docs/static/evo-core.css

```diff
@@ -1,4 +1,6 @@
 :root {
+    --border-radius-50: 8px;
```

Comment:
Why is `50` set to `8px`? Feels a bit arbitrary.

Reply:
Using 100 and 50 leaves plenty of space to slot in other values if we need to. Going with border-radius-medium and border-radius-small, or border-radius-2 and border-radius-1 leaves no room and makes naming difficult (we've actually used both of those systems in the past before and they didn't scale well for those reasons).